server/settings/main.py

```diff
@@ -71,6 +71,24 @@ def _fallback_ocio_config_profile_types():
 
 def _ocio_built_in_paths():
     return [
+        {
```

Comment:
Is the order ok? ACES 1.3, ACES 2.0, ACES 1.2, Nuke

Reply:
I would say that it should be: ACES 2.0, ACES 1.3, ACES 1.2, Nuke